spec/swagger.yaml

```diff
@@ -6586,6 +6590,232 @@ paths:
           source: |
             curl -H "Authorization: Bearer $smartlingToken" -H "Content-Type: application/json" https://api.smartling.com/vendors-api/v2/agencies/{agencyUid}/jobs
 
+#
+# Reports API
+#
+  '/reports-api/v3/word-count':
+    get:
+      description: >
+        Word Count report data for the specified parameters.
+      summary: Get Word Count Report
+      tags:
+      - Reports
+      parameters:
+      - name: startDate
+        in: query
+        type: string
+        required: true
+        format: YYYY-MM-DD
+        description: A parameter to limit words processed on or after ```startDate``` in UTC timezone.
+      - name: endDate
+        in: query
+        type: string
+        required: true
+        format: YYYY-MM-DD
+        description: A parameter to limit words processed on or before ```endDate``` in UTC timezone.
+      - name: accountUid
+        in: query
+        type: string
+        required: true
+        description: Unique identifier of corresponding account.
+      - name: projectIds
+        in: query
+        type: array
+        items:
+          type: string
+        required: false
+        description: Optional parameter to filter to the specified list of projects.
+      - name: jobUids
+        in: query
+        type: array
+        items:
+          type: string
+        required: false
+        description: Optional parameter to filter to the specified list of translation jobs.
+      - name: userUids
+        in: query
+        type: array
+        items:
+          type: string
+        required: false
+        description: Optional parameter to filter to the specified list of translation resources.
+      - name: targetLocaleIds
+        in: query
+        type: array
+        items:
+          type: string
+        required: false
+        description: Optional parameter to filter to the specified list of translation locale IDs.
+      - name: includeTranslationResource
+        in: query
+        type: boolean
+        required: false
+        default: true
+        description: >
```

Comment:
`>` seems unnecessary. Do wee need  it?

Reply:
It's an instruction that the description is a multiline strings